docs/sql-migration-guide.md

```diff
@@ -222,6 +222,8 @@ license: |
 
   - Since Spark 3.0, when casting interval values to string type, there is no "interval" prefix, e.g. `1 days 2 hours`. In Spark version 2.4 and earlier, the string contains the "interval" prefix like `interval 1 days 2 hours`.
 
+  - Since Spark 3.0, day-time interval strings are converted to intervals with respect to the `from` and `to` bounds. If an input string does not match to the pattern defined by specified bounds, the `ParseException` exception is thrown. In Spark version 2.4, the `from` bound was not taken into account, and the `to` bound was used to truncate the resulted interval. To restore the behavior before Spark 3.0, you can set `spark.sql.legacy.fromDayTimeString.enabled` to `true`.
```

Comment:
it's better to give an example ad show the different results between 3.0 and 2.4